platform_versioned_docs/version-24.1/enterprise/kubernetes-minimal-install.mdx

```diff
@@ -0,0 +1,418 @@
+---
+title: Install Seqera Platform on Kubernetes
+description: Deploy Seqera Platform Enterprise with Kubernetes
+date: "11 Sep 2024"
+tags: [kubernetes, deployment]
+---
+
+This installation guide describes how to install Seqera Platform on a Kubernetes cluster.
+
+## Prerequisites
+
+The following prerequisites are required to deploy Seqera on a Kubernetes cluster.
+
+- A Kubernetes cluster version 1.30.2 or newer
```

Comment:
We currently run 1.28 in our environments , pinning to a version may result in this doc being dated. 

What about `- A Kubernetes cluster running a supported version`